windows/src/desktop/kmshell/xml/config.css

```diff
@@ -43,7 +44,8 @@
   bottom: 0;
   width: 100%;
   overflow: hidden;
-  background: #79C3DA;
+  /*TODO background: #79C3DA;*/

```

Comment:
Is this temporary code that you want to put back before merging? Or why different background colors from before?

Reply:
When reviewing it was suggested to remove the blue background. In doing that it looked a bit too washed out. I then tried the grey defined in the `Keyman Design Approach Manual`. The reason I left the comment there was because I was still unsure which way to go. The grey looks ok in my opinion a bit dull but closer the Windows OS feel. Shawn had commented that UI which looks vastly different from the native feel can give the user are feeling of less trust in the application.  What are your thoughts?


![blue](https://user-images.githubusercontent.com/58423624/192409303-4060fa4b-346a-4179-8dff-ff55c84f3ac8.png)
![default](https://user-images.githubusercontent.com/58423624/192409316-b32011e9-5d7f-45b8-a1a3-8fa04e01a59c.png)
![grey](https://user-images.githubusercontent.com/58423624/192409324-ef9345df-22ac-495e-815e-5bebdc7d02d8.png)

Reply:
My pref would be either white bg or blue bg. Buttons look a little too 'disabled' however. May need to rethink the colour of those a little?

Reply:
We have trouble with matching native UI -- it changes so much that we can't keep up. So I think we don't try and keep up. AFAICT, most modern apps have their own UI themes anyway so we aren't losing much...

Reply:
Sticking with the blue